lighthouse-core/lib/page-functions.js

```diff
@@ -508,7 +509,7 @@ function getNodeDetails(element) {
     selector: getNodeSelector(element),
```

Comment:
reuse `selector` here instead of recomputing? :)

Reply:
definitely agree :)